docker/README.md

```diff
@@ -0,0 +1,98 @@
+**This documentation focuses on the Deployer Docker image and its usage, not the Deployer itself. If you want learn more about the Deployer, please read [its documentation](../packages/deployer/README.md).**
+
+# Build
+You can build the Docker image by running:
+```bash
+docker build -f docker/Dockerfile -t api3/deployer:latest .
+```
+
+Before the build, all Deployer dependencies (but not the Deployer itself) must be available in the package registry.
+
+You can use build argument `yarnRegistry` to specify a different package registry for package installation. This is useful for testing a Docker build with unreleased dependencies. You can use for example [Verdaccio](https://verdaccio.org/) for this purpose.
+
+# Configuration
+## Credentials
+In order to deploy Airnode to cloud you need to pass your cloud credentials to the container. We're currently supporting only AWS cloud.
+
+### AWS
+You have two options how to pass your AWS credentials
+
+#### Via environment variables:
+You can either prepare a file with environment variables and pass that
+```bash
+$ cat aws.env
+AWS_ACCESS_KEY_ID=...
+AWS_SECRET_ACCESS_KEY=...
+AWS_SESSION_TOKEN=...
+
+$ docker run --env-file aws.env ...
+```
+
+or you can pass the environment variables separately
+```bash
+docker run -e AWS_ACCESS_KEY_ID=... -e AWS_SECRET_ACCESS_KEY=... ...
+```
+
+#### Via configuration file:
+You can use a [credential file](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html#cli-configure-files-where) to store toy AWS credentials. Passing this file to the container will allow Deployer to use these credentials:
+```bash
+docker run -v ${HOME}/.aws/credentials:/root/.aws/credentials ...
+```
+
+## User mapping
+Since Deployer provides an output in a form of a `receipt.json` file, you may find easier to run the Deployer CLI with the UID/GID you're using outside the container to avoid any permission problems. For this reason Deployer image provides environment variables `USER_ID` and `GROUP_ID` allowing you to set UID/GID for execution:
+```bash
+docker run -e USER_ID=$(id -u) -e GROUP_ID=$(id -g) ...
```

Comment:
Can't this be a default somehow?

Reply:
It can't really :confused: It has to be passed from outside (the `id` command is run on your underlying system) and it has to be passed when the container is starting.

You can run it without it but the `receipt.json` will be written to your filesystem by `root` which is usually not what you want. I mean, it doesn't break anything but it's anoying.